intermediate-week-1/controllers/user/getUserInfo.js

```diff
@@ -8,6 +10,17 @@ const getUserInfo = async (req, res, next) => {
    *      email: *****************
    * }
    */
+   try {
```

Comment:
Nice attempt. 

The user details are to come from the authentication middleware after verification though. 

Look through the week-1-solution branch to see an example of how that works.